docs/progress.md

```diff
@@ -46,7 +46,7 @@ Table columns:
 | 🚧  | [Toggle](https://developer.apple.com/documentation/swiftui/toggle)           |     |
 | 🚧  | [Picker](https://developer.apple.com/documentation/swiftui/picker)           |     |
 |     | [DatePicker](https://developer.apple.com/documentation/swiftui/datepicker)   |     |
-|     | [Slider](https://developer.apple.com/documentation/swiftui/slider)           |     |
+| ✅  | [Slider](https://developer.apple.com/documentation/swiftui/slider)           |     |
```

Comment:
Shouldn't this be 🚧 with the unresolved FIXME in the comment at `onEditingChanged`?